### ISSUE TITLE
Cannot debug unit test, the connection seems to lost when opened in chrome

### ISSUE THREAD

**Issue by anuragvohraec** [denoland/deno#4778]
Hi Team,

Here is my code: index.ts
```js
export function addAndSquare(a:number, b:number){
    let x= a+b;
    let r = Math.pow(x,2);
    return r;
}
```
And below is the test case for it :
```js
import { addAndSquare } from './index.ts';
import { assert, assertEquals } from 'https://deno.land/std/testing/asserts.ts';
const {test} = Deno;

test("Test AddAndSquare function",()=>{
    let a  = addAndSquare(4,5);
    assertEquals(a,81);
})
```

I was trying to debug this code to learn how to debug code with deno.
So I ran following commands:
`deno test --inspect-brk`
And open chrome://inspect tools, to makr debug points and debug this code.
My First break point is in index.tx at line `let r = Math.pow(x,2);` and second debug point is in index_test.ts  at line ` assertEquals(a,81);`, the debugger never let it reach assertEqual line, how so many time I press F8 in chrome debugger.

Am I doing something wrong ?
How do we debug unit test codes [one without servers] in deno ?

**Comment by bartlomieju** [comment]
Hi @anuragvohraec, can you try again with latest version of Deno? We've made several fixes to inspector just before releasing 1.0 so hopefully this issue is no longer relevant.

**Comment by anuragvohraec** [comment]
>  hopefully this issue is no longer relevant.

@bartlomieju yes all good, we can close this issue now.
